tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -640,6 +626,69 @@ def _run(self):
                         pcnt_used=0
                     ).save()
 
+                # Invoke the aggregated sync flow
+                if "provisioner/%s" % NS.tendrl_context.integration_id \
+                    in NS.node_context.tags:
+                    params = {}
```

Comment:
I dont think we need a job for this. Simply run this code if "provisioner/$integration_id" tag is present